File: Part4/Fortran/Function_IgnoredReturn.f90
program function_ignored_return
    implicit none
    character( len=100 ) :: user_input
    integer :: entered_integer, ios
    
    write( *, * ) 'Type 1 and press enter.'
    read( *, '(A)' ) user_input
    
    ! Function called but return value ignored
    read( user_input, *, iostat=ios )
    
    write( *, * ) 'The user entered the integer ', entered_integer
    
end program function_ignored_return

File: Part4/Rust/Function_IgnoredReturn.rs
use std::io;

fn main()
{
    let mut user_input = String::new();
    let entered_integer: i32;
    
    println!( "Type 1 and press enter." );
    io::stdin().read_line( &mut user_input ).expect( "Failed to read line" );
    
    // Function called but return value ignored
    user_input.trim().parse::<i32>();
    
    println!( "The user entered the integer {}", entered_integer );
}
